### ISSUE TITLE
Suddenly I get "Can't move message"error

### ISSUE THREAD

**Issue by cpranzl** [pierre-alain-b/rainloop-nextcloud#55]
Suddenly and without any hint why I got problems with moving messages to folders . It started when I sent a mail and got the error 

![image](https://user-images.githubusercontent.com/3177622/37776698-294e702e-2de6-11e8-893d-c0e21b6dc0ad.png)

I was puzzled but had no time to analyze this issue further so I checked with the recipient of the mail if the mail was sent, which he acknowledged.

Later that day I experienced what I think is the same error, when I tried to delete a mail.

![image](https://user-images.githubusercontent.com/3177622/37777007-da08e2b4-2de6-11e8-9415-9434c3827d70.png)

It translates to "On <mynextcloud> following is displayed Can't move message".

**Comment by cpranzl** [comment]
Nextcloud quota for my user is unlimited and I've 33GB free space.
I've activated logging in applications.ini. Any hint for what I'm looking?

**Comment by cpranzl** [comment]
I'm sooooo dumb! I"m using 103% of my 2GB!

**Comment by pierre-alain-b** [comment]
Don't blame yourself too hard, we all did it once to forget about a RAM-limited situation :-)

**Comment by cpranzl** [comment]
Maybe I should clearify my problems and how I solved it.

**Problem:** Couldn't move messages into folders, and since deleting is just moving them into trash, I couldn't delete messages either.

**Steps taken:**
1. Checked disk usage (33GB free)
2. Checked quota of my nextcloud-account (which is the one that is unlimited)
3. Activated logging in rainloop-nextcloud and found messages regarding a quota
4. Got enlightenment that dovecot quota is meant and not nextcloud quota

**Solution:** Changed in mySQL-DB mail-account quota from 2048 to 4096
